tests-e2e/global-setup.ts

```diff
@@ -1,157 +1,16 @@
-}
-
-const buildSite = (name: string, destInTmp: string, configPath: string) => {
-  return hugo(
-    {
-      themesDir:   fromRoot("./"),
-      destination: fromTmp("dist", destInTmp),
-      cacheDir:    fromTmp("hugo_cache"),
-      config:      configPath,
-      baseURL:     destInTmp,
-      verbose:     true
-    },
-    {
-      cwd: fromRoot(`./test-sites/${name}`),
-      env: {
-        ...process.env,
-        ...stringifyEnv(testEnv)
-      }
-    }
-  )
-}
-
-/**
- * Redirect all requests /path/to/thing to /ocw-ci-test-www/path/to/thing,
- * except requests to /courses/.
- */
-const OCW_WWW_REDIRECT: RedirectionRule = {
-  type:      "rewrite",
-  match:     /^\/(?!(courses\/))/,
-  transform: url => `/ocw-ci-test-www${url}`
-}
-
-const steps = {
-  /**
-   * Set up a sserver that:
-   *  - serves the contents of `test-sites/__fixtures__`
-   *  - redirects requests like /not/a/course/page to /ocw-ci-test-www/not/a/course/page
-   */
-  setupStaticApiFixtures: (): void => {
-    const server = new SimpleServer(
-      (request, response) => {
-        return handler(request, response, {
-          public: fromRoot(`./test-sites/__fixtures__`)
-        })
-      },
-      {
-        rules: [OCW_WWW_REDIRECT]
-      }
-    )
-    server.listen(STATIC_API_PORT)
-  },
-  /**
-   * Build all sites with Hugo.
-   *
-   * The output will go in `test-sites/tmp/dist`.
-   */
-  buildAllSites: async (): Promise<void> => {
-    await Promise.all(
-      Object.entries(TEST_SITES).map(([alias, site]) => {
-        const destInTmp =
-          alias === "www" ? `/${site.name}` : `/courses/${site.name}`
-        return buildSite(site.name, destInTmp, site.configPath)
-      })
-    )
-  },
-  /**
-   * Set up a server that:
-   *  - serves the contents of `test-sites/tmp/dist`
-   *  - redirects requests like /not/a/course/page to /ocw-ci-test-www/not/a/course/page
-   */
-  serveSites: (): void => {
-    const server = new SimpleServer(
-      (request, response) => {
-        return handler(request, response, {
-          public: fromTmp("dist")
-        })
-      },
-      {
-        rules: [
-          {
-            type:      "redirect",
-            match:     /^\/static\//,
-            transform: url => `http://localhost:${env.WEBPACK_PORT}${url}`
-          },
-          OCW_WWW_REDIRECT
-        ]
-      }
-    )
-    server.listen(LOCAL_OCW_PORT)
-  },
-  /**
-   * Pretty-print a table to the terminal with the URLs of all the sites.
-   */
-  announceSites: (): void => {
-    const table = new Table({
-      head:  ["Site", "URL"],
-      style: { head: ["cyan", "bold"] }
-    })
-    Object.entries(TEST_SITES).forEach(([alias, site]) => {
-      table.push([site.name, siteUrl(alias as TestSiteAlias)])
-    })
-    console.log(
-      [
-        "\n",
-        color.bold("Now serving the following sites:\n"),
-        table.toString(),
-        "\n"
-      ].join("")
-    )
-  }
-}
+import LocalOCW, { fromRoot } from "./LocalOcw"
```

Comment:
Re PascalCase:
1. PascalCase is common for classes. (And also for react components, even though they are more commonly functions now adays).
2. It's common for filenames to have the same name as their default export *I probably picked up the convention from https://github.com/airbnb/javascript#naming--filename-matches-export)

> ... so we don't have a mix of both because that's confusing.

We do generally follow the two rules above. It's most obvious in our react code, because that's where we use default exports the most. The kebab-cased js files that we have do strike me as a little odd.


BTW: The above applies to JS or TS equally. (Two TS-specific places where PascalCase is common are the names of types and the names of enum members).